roles/confluent.test/molecule/plain-rhel/molecule.yml

```diff
@@ -99,6 +123,11 @@ provisioner:
     group_vars:
       all:
         sasl_protocol: plain
+
+        # ssl_enabled: true
```

Comment:
remove commented out code, appears changes were made to this scenario for dev, probably revert